### ISSUE TITLE
Correct order of heading elements

### ISSUE THREAD

**Issue by schalkneethling** [mdn/yari#1969]
The `Metadata` heading should be an `h3` as opposed to an `h4` to ensure sequential heading order.

```
Heading elements are not in a sequentially-descending order
```

**Comment by schalkneethling** [comment]
Will be addressed as part of https://github.com/mdn/yari/issues/1613
